Help Desk process

```diff
@@ -0,0 +1,31 @@
+Process for Help Desk Requests from External Users
+
+When a new issue is received from an external user, it automatically creates a JIRA ticket.
+Help Desk managers send a generic response (within one day) to the user notifying them that we will get them an answer soon
```

Comment:
Annoyingly, JIRA doesn't (as far as I know) have a way to automate replies. However, once we stop allowing help desk requests to come through help@kbase.us and insist that users contact us via the web form, it seems like we should be able to add a script that makes the web form send the user a note saying "Thanks for contacting us; someone will get back to you soon."

It's not urgent, since at this point we get few enough help desk requests that a human has no trouble handling them.